web-app/HeimScripts/heatmap/run.R

```diff
@@ -450,6 +450,7 @@ dendrogramToJSON <- function(d) {
 
 
 addClusteringOutput <- function(jsn, measurements) {
+  measurements[ is.na(measurements) ] <- 0  # All NAs to zero. Only for purpose of clustering
```

Comment:
The other (possibly better) option is to drop rows with less than 3 values at the beginning of the script. Replacing NAs with zeros here is probably not proper.